docs/DeveloperGuide.md

```diff
@@ -3,9 +3,6 @@ layout: page
 title: Developer Guide
 ---
 
-* Table of Contents
-  {:toc}
```

Comment:
I think we might need to create a table of contents and fill it up when our DG is done so our peer reviewers can navigate the document easily. What do you think?

Reply:
Sure, that would be better I think.

Reply:
Sure, that would be better.